packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import { PlusSquare } from 'react-feather';
+
+import { useAddressBookPage } from '@magento/peregrine/lib/talons/AddressBookPage/useAddressBookPage';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import { GET_CUSTOMER_ADDRESSES } from '../CheckoutPage/AddressBook/addressBook.gql';
+import AddressCard from '../CheckoutPage/AddressBook/addressCard';
+import Icon from '../Icon';
+import LinkButton from '../LinkButton';
+import { Title } from '../Head';
+import defaultClasses from './addressBookPage.css';
+
+const PAGE_TITLE = `Address Book`;
+
+const AddressBookPage = props => {
+    const talonProps = useAddressBookPage({
+        queries: {
+            getCustomerAddressesQuery: GET_CUSTOMER_ADDRESSES
+        }
+    });
+    const { customerAddresses, handleAddAddress } = talonProps;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const addressBookElements = useMemo(() => {
+        return customerAddresses.map(addressEntry => (
+            <AddressCard key={addressEntry.id} address={addressEntry} />
```

Comment:
`onSelection`, `onEdit`, and `isSelected` props all threw errors because they are required.